BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -93,12 +93,8 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     walletView.headerView.addFundsButton.addTarget(self, action: #selector(tappedAddFunds), for: .touchUpInside)
     walletView.headerView.settingsButton.addTarget(self, action: #selector(tappedSettings), for: .touchUpInside)
     walletView.headerView.grantsButton.addTarget(self, action: #selector(tappedGrantsButton), for: .touchUpInside)
-    
-    walletView.headerView.setWalletBalance(
```

Comment:
We should update UI here with old data, otherwise it'll be empty until balance comes in